automation/taskcluster/decision_task.py

```diff
@@ -73,6 +73,8 @@ def pr():
     ):
         other_tasks[taskcluster.slugId()] = craft_function()
 
+    other_tasks[taskcluster.slugId()] = BUILDER.craft_dependencies_task()
```

Comment:
Adding this to the PR task so it can be tested before merging...  Should remove this before merging into master.